patches/chrome-android-java-res-layout-toolbar_phone.xml.patch

```diff
@@ -1,12 +1,21 @@
 diff --git a/chrome/android/java/res/layout/toolbar_phone.xml b/chrome/android/java/res/layout/toolbar_phone.xml
-index 62a3a3f3dc36d6417519832e987466ff26a18ea2..e4cdbe17da6b3db357992f755aa884b1f026a7a8 100644
+index 62a3a3f3dc36d6417519832e987466ff26a18ea2..367cf73d33de0abcd0ec076f9faa5f07c57e2536 100644
 --- a/chrome/android/java/res/layout/toolbar_phone.xml
 +++ b/chrome/android/java/res/layout/toolbar_phone.xml
+@@ -31,7 +31,7 @@
+         android:orientation="horizontal"
+         android:layout_width="wrap_content"
+         android:layout_height="wrap_content"
+-        android:layout_gravity="top|end" >
++        android:layout_gravity="top|end" android:layout_marginEnd="8dp" >
```

Comment:
can you put `android:layout_marginEnd="8dp"` in the line above this one so it's just an addition and not a change to a line

Reply:
done